### PR TITLE
Update Chapel versions to 2.2 pre-release

### PR DESCRIPTION
**Pull request by tzinsky**
Update version numbers for the next Chapel release and ran printstuff to verify results: 

`[Test Summary - 240626.154334]
[Error matching compiler output for compflags/bradc/printstuff/version]
[Error matching compiler output for compflags/bradc/printstuff/zall]
[Error matching compiler output for compflags/bradc/printstuff/zcopyvers]
[Error matching compiler output for compflags/bradc/printstuff/zhelpvers]
[Error matching compiler output for compflags/bradc/printstuff/zlicevers]
[Summary: #Successes = 5 | #Failures = 5 | #Futures = 0 | #Warnings = 0 ]
[Summary: #Passing Suppressions = 0 | #Passing Futures = 0 ]
[END]`

New test run after updating versions and removing LF at the end of a file: 

`[Test Summary - 240626.174407]
[Summary: #Successes = 10 | #Failures = 0 | #Futures = 0 | #Warnings = 0 ]
[Summary: #Passing Suppressions = 0 | #Passing Futures = 0 ]
[END]`

### CODE REVIEW

File: man/confchpl.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-:Version: 2.1
+:Version: 2.2.0 pre-release
```
Comment (riftEmber) [inline man/confchpl.rst:2]: Should this still have just major and minor version numbers like `2.1` did?

File: man/confchpldoc.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-:Version: 2.1 pre-release
+:Version: 2.2.0 pre-release
```
Comment (riftEmber) [inline man/confchpldoc.rst:2]: ditto above

**Comment by riftEmber** [comment]
Linking previous release's PR for this for reference: https://github.com/chapel-lang/chapel/pull/24646

File: man/confchpldoc.rst (hunk 2 above)
Comment (tzinsky) [reply man/confchpldoc.rst:2]: Ditto :D

File: man/confchpl.rst (hunk 1 above)
Comment (tzinsky) [reply man/confchpl.rst:2]: Yes, will fix...

File: test/compflags/bradc/printstuff/version.goodstart [hunk 3]
```diff
@@ -1 +1 @@
- version 2.1.0
\ No newline at end of file
+ version 2.2.0
```
Comment (arezaii) [inline test/compflags/bradc/printstuff/version.goodstart:1]: I think you need to remove the trailing line break here to get the tests to pass

Comment (tzinsky) [reply test/compflags/bradc/printstuff/version.goodstart]: Yes, that is correct and I could only get this working in vi...    

```
[Test Summary - 240626.174407]
[Summary: #Successes = 10 | #Failures = 0 | #Futures = 0 | #Warnings = 0 ]
[Summary: #Passing Suppressions = 0 | #Passing Futures = 0 ]
[END]
```

File: man/confchpldoc.rst [hunk 4]
```diff
@@ -1,5 +1,5 @@
 
-:Version: 2.1 pre-release
```
Comment (arezaii) [inline man/confchpldoc.rst:2]: eek, this is my bug in the release that it still says `pre-release` in the `chpldoc` man page.

Comment (tzinsky) [reply man/confchpldoc.rst:2]: Ugh, do we need to respin?

**Pull request merged**